earthpy/tests/test_spatial.py

```diff
@@ -48,44 +41,39 @@ def test_extent_to_json():
     with pytest.raises(AssertionError):
```

Comment:
if we do the above suggestion this would then just be `extent_to_json` (without the `es.`)